### PR TITLE
Feature/organisations form

### PR DESCRIPTION
**Pull request by Bluesmile82**
[ORC-231 - Upload new organisations (without projects)](https://vizzuality.atlassian.net/browse/ORC-231)
[ORC-280- Update styles of button to access the form](https://vizzuality.atlassian.net/browse/ORC-280)
[ORC-281 - Implement the organisations form](https://vizzuality.atlassian.net/browse/ORC-281)
including:
- Errors
- Thank you page

The associated projects part will be added in a new PR. 

Also:
- Add color border tops to the nav on the HOME page
- Update the network list to include filter for the published_status so that we only display the accepted or null networks

### CODE REVIEW

**Review by clementprdhomme — COMMENTED** [review]
In addition to my inline comments:

- The input error styles should be different:
  - Use `red-800` instead of the default `red-500`
  - The label shouldn't change colour
  - The input's border should be red
- Can we display a scrollbar inside the dropdowns (`Select`)?
- Could you review with Lidia if we couldn't  make the header sticky or else add an extra submit button at the bottom of the form? I fear that some user won't understand that they have to scroll at the top again to submit.
  - In both cases, if there is a general error and the user has scrolled down, they might not see the error message so maybe we should automatically scroll to the top of the page when that happens.
- Don't we need to save changes to the repository to allow public users to create new organisations (Strapi's “User & permissions” plugin)?

**Comment by Bluesmile82** [comment]
> Don't we need to save changes to the repository to allow public users to create new organisations (Strapi's “User & permissions” plugin)?
It seems this is already active

**Comment by Bluesmile82** [comment]
Ready again! Thank you so much for the review!

File: client/src/app/(modules)/network/(form)/new/organisation/page.tsx [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+  const router = useRouter();
+
+  if (!hasData || !fields) {
+    return null;
+  }
+
+  const onSubmit: SubmitHandler<z.infer<typeof formSchema>> = (data) => {
+    const normalizedData = {
+      // ...data,
```
Comment (clementprdhomme) [inline client/src/app/(modules)/network/(form)/new/organisation/page.tsx:210]: Uncomment this line.

File: client/src/app/(modules)/network/(form)/new/organisation/page.tsx [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+      url: data.url && data.url.startsWith('http') ? data.url : `https://${data.url}`,
```
Comment (clementprdhomme) [inline client/src/app/(modules)/network/(form)/new/organisation/page.tsx:211]: If `data.url` is `undefined`, then you're sending `'https://undefined'` to the server.

File: client/src/hooks/networks/index.ts [hunk 3]
```diff
@@ -90,6 +90,15 @@ export enum NetworkProjectStatusFilter {
   NotStarted,
 }
 
+const ACCEPTED_STATUS_FILTER = {
+  $or: [
+    {
+      publication_status: { $eq: 'accepted' },
+    },
+    { publication_status: { $null: true } },
```
[the remark this answers is not in the document]
Comment (Bluesmile82) [reply client/src/hooks/networks/index.ts:98]: Its status will be _proposed_. We don't want to show proposed networks on the list until they are accepted

[the remark this answers is not in the document]
Comment (clementprdhomme) [reply client/src/hooks/networks/index.ts:98]: I agree with you. But why should we show organisations that don't have any status? I think we should only show the ones that have an `'accepted'` status.

**Review by clementprdhomme — COMMENTED** [review]
I think there's one more change needed:

- When you submit an empty form, there are two things that should happen:
  - The invalid fields should have a red border (I don't see them on Firefox neither on Chrome)
  - If before submitting, you've selected “Other” in the “Organisation type” field, then the “Organisation type: Other” field doesn't receive a validation error. The error only appears once all the other validation error have been solved.

**Pull request merged**